project/ProjectPlugin.scala

```diff
@@ -25,23 +25,23 @@ object ProjectPlugin extends AutoPlugin {
 
     lazy val micrositeSettings: Seq[Def.Setting[_]] = Seq(
       micrositeName := "Fetch",
-      micrositeDescription := "Simple & Efficient data access for Scala and Scala.js",
+      micrositeDescription := "Simple & Efficient data for Scala",
```

Comment:
:thinking:

Reply:
That is the description that appears in the new redesign. I´ll make sure it will be that way.